api/src/Entity/User.php

```diff
@@ -135,7 +135,7 @@ class User extends BaseEntity implements UserInterface, PasswordAuthenticatedUse
         ]
     )]
     #[Groups(['create'])]
-    #[ORM\OneToOne(targetEntity: Profile::class, inversedBy: 'user', cascade: ['persist'])]
+    #[ORM\OneToOne(targetEntity: Profile::class, inversedBy: 'user', cascade: ['persist'], fetch: 'EAGER')]
```

Comment:
This was a small general performance improvement: With every call User is loaded and then Profile for access verification. The latter is because of this line: https://github.com/ecamp/ecamp3/blob/devel/api/src/Entity/User.php#L208 (roles stored on profile entity)

Eager loading combines both entities in 1 DB query instead of 2 separate ones.